ui/src/app/accusoft-editor.component.js

```diff
@@ -2,20 +2,20 @@ import React, { useRef, useEffect, useState } from 'react'
 
 export default function Editor(props) {
   const container = useRef()
-  const [hasRun, setHasRun] = useState()
-  const editor = new PrizmDocEditor({
-    baseUrl: 'https://prizmdoc-integ.canopy.ninja/',
-    sessionId: props.sessionId
-  });
 
   useEffect(() => {
-    if (container.current && !hasRun) {
-      setHasRun(true)
+    if (props.sessionId) {
+      container.current.innerText = ''
+      const editor = new PrizmDocEditor({
+        baseUrl: 'https://prizmdoc-integ.canopy.ninja/',
+        sessionId: props.sessionId
+      });
       editor.embed({
         container: container.current
       }).then(editor => props.setEditor(editor))
     }
-  }, [container.current, hasRun])
```

Comment:
dan abramov seems to indicate no need to include ref to dom node in deps array and in fact it was causing problems for us